### ISSUE TITLE
Submit to pypi

### ISSUE THREAD

**Issue by nsheff** [databio/logmuse#1]
When stable we will need to put this on pypi.

**Comment by vreuter** [comment]
https://pypi.org/project/logmuse/

**Comment by nsheff** [comment]
wow, didn't realize you can have version 0.0! cool.

Maybe missing MANIFEST.in?

```
pip3 install --user logmuse
Collecting logmuse
  Downloading https://files.pythonhosted.org/packages/0f/ea/36987984c280ae784ab74c532f3322160b3c36ff7fec3b10669398bdd9e7/logmuse-0.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-4c_fg3dt/logmuse/setup.py", line 22, in <module>
        DEPENDENCIES = read_reqs("all")
      File "/tmp/pip-install-4c_fg3dt/logmuse/setup.py", line 12, in read_reqs
        with open(depsfile, 'r') as f:
    FileNotFoundError: [Errno 2] No such file or directory: 'requirements/requirements-all.txt'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-4c_fg3dt/logmuse/
```

**Comment by nsheff** [comment]
fixed in 2ab306d2d9461187dc6eac858d227cbf87cbe48b

**Comment by vreuter** [comment]
> fixed in [2ab306d](https://github.com/databio/logmuse/commit/2ab306d2d9461187dc6eac858d227cbf87cbe48b)

great thanks
